docs/guidelines.md

```diff
@@ -27,7 +27,7 @@ those requirements before an approval.
 
 Developers will follow the [GitHub
 Flow](https://guides.github.com/introduction/flow/) model for PR submissions.
-Howeber, Ocopea branching is not allowed.  GitHub supports cloning a repository
```

Comment:
I kinda want Howeber to be a word...

Reply:
Thought I fixed that...

Reply:
yes, see a later commit
tried to squash but failed to do so (hence 3 commits)
howeber, does it automatically squash when merging?